docs/user/getting_started_renkulab.rst

```diff
@@ -0,0 +1,81 @@
+.. _getting_started_renkulab:
+
+Getting Started on `renkulab.io <https://renkulab.io>`__
+========================================================
+
+This tutorial will help you get started working on the Renku platform deployed
+at `renkulab.io <https://renkulab.io>`__, it will teach you how to use Renku to:
+
+1. `Create a new project`_
+2. `Add input data to your project`_
+3. Share your results and collaborate with your peers
+
+Create a new project
+^^^^^^^^^^^^^^^^^^^^
+
+First, head to `renkulab.io <https://renkulab.io>`__ and click on the **Login**
+button located on the top right corner of the Renku web interface.
+
+You can sign in using with your GitHub or LinkedIn account by
```

Comment:
why linkedin? I'd rather see e.g. gitlab, google...

Reply:
GitLab does not have real name of users. Google, yes but we need a privacy policy document on a URL.